Gemfile

```diff
@@ -3,3 +3,7 @@
 source 'https://rubygems.org'
 
 gemspec
+
+group :test do
+  gem 'coveralls', '~> 0.8.2'
```

Comment:
Do we need this? I do not use coveralls for myself, nor I think devtools supports this.